components/image.css

```diff
@@ -0,0 +1,7 @@
+.img-responsive{
```

Comment:
This is not the correct css for responsive image. You should read some more about responsive image then apply the necessary css.